src-tauri/src/mod_management/mod.rs

```diff
@@ -679,14 +679,22 @@ pub fn delete_northstar_mod(game_install: GameInstall, nsmod_name: String) -> Re
     }
 
     // Get installed mods
-    let installed_ns_mods = get_installed_mods_and_properties(game_install)?;
+    let installed_ns_mods = get_installed_mods_and_properties(game_install.clone())?;
 
     // Get folder name based on northstarmods
     for installed_ns_mod in installed_ns_mods {
         // Installed mod matches specified mod
         if installed_ns_mod.name == nsmod_name {
             // Delete folder
-            return delete_mod_folder(&installed_ns_mod.directory);
+            return match delete_thunderstore_mod(
```

Comment:
This assumes that we always want to delete all Northstar mods from a given Thunderstore package. This might not necessarily reflect what the user actually wants.

Maybe they want [GlitchOverhaul](https://northstar.thunderstore.io/package/S2Mods/GlitchOverhaul/) but only some parts of it for example...

(I swear the whole Northstar mod vs Thunderstore mod thing is such a mess. We should've just switched the mod format when we migrated to Thunderstore x_x)